packages/@sfcc-core/apiconfig/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "current"
```

Comment:
The TypeScript builds are setup to actually use babel, and specifically to compile to the "current" version of NodeJS. So if we change our CI pipeline to use a different version of node and publish to NPM, we can automatically support new versions of node.